kolibri/logger/management/commands/userdata_utils.py

```diff
@@ -0,0 +1,327 @@
+from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
Maybe let's move this into `logger/utils` - so as to avoid having non-management commands in the commands folder.